JS/Lesson33/gulcin/my-app/src/modules/list/ListProvider.jsx

```diff
@@ -0,0 +1,54 @@
+import { createContext, useReducer, useContext } from "react";
+
+
+export const todoData = [
+    {id: 1, title: "Do dishes", completed: false},
+    {id: 2, title: "Do homework", completed: false},
+    {id: 3, title: "Go running", completed: false},
+    {id: 4, title: "Do dishes 2", completed: false},
+    {id: 5, title: "Do homework 3", completed: false},
+    {id: 6, title: "Go running 4", completed: false},
+  ];
+
+export const TODOLIST_ACTIONS = {
+   toDoCompleted: 'todo_completed',
```

Comment:
I think name 'complete' would be better aligned with the other action type ('delete')